plugins/misp/plugin.spec.yaml

```diff
@@ -157,18 +170,22 @@ types:
     event:
       title: Event ID
       description: Event ID to append to
+      example: "1"
       type: string
     text:
       title: Text
       description: Text
+      example: "Example text"
       type: string
     comment:
       title: Comment
       description: Attribute comment
+      example: "Example attribute comment"
       type: string
     distribution:
       title: Distribution
       description: Distribution type
+      example: "All Communities"
```

Comment:
Same as above, when there is an enum - shouldn't the example be enum value?

Reply:
Line 194, there is an enum like `All Communities`